src/test/java/com/google/firebase/messaging/FirebaseMessagingTest.java

```diff
@@ -97,7 +97,7 @@ public void testNoProjectId() {
   }
 
   @Test
-  public void testNullMessage() throws Exception {
```

Comment:
did you remove the throws on purpose? (I'm good either way, but the next test has it)

Reply:
Yes, the compiler flagged this as redundant. Other tests call methods that throw checked exceptions.